courseScheduleII.py

```diff
@@ -7,9 +7,9 @@ def findOrder(self, numCourses, prerequisites):
         """
         summary = {}
         degree = [0] * numCourses
-        for pair in prerequisites:
-            summary[pair[1]] = summary.get(pair[1], []) + [pair[0]]
-            degree[pair[0]] += 1
+        for x, y in prerequisites:
```

Comment:
When I said `x` and `y`, I don't mean you should use `x` and `y` as variable names...